tests/conftest.py

```diff
@@ -680,6 +680,38 @@ def strict_pre_save(self, model_instance, add):
     DateField.pre_save = strict_pre_save
 
 
+@pytest.fixture(autouse=True, scope="session")
+def detect_missing_auto_now_in_update_fields():
+    from django.apps import apps
+    from django.db.models import Model
+
+    auto_now_fields = {}
+    for model in apps.get_models():
+        for field in model._meta.get_fields():
+            if getattr(field, "auto_now", False):
+                auto_now_fields.setdefault(model._meta.label, []).append(field.name)
+
+    original_save = Model.save
+
+    def strict_save(
+        self,
+        *args,
+        force_insert=False,
+        force_update=False,
+        using=None,
+        update_fields=None,
```

Comment:
Qui prend les paris pour dans combien de version Django ajoute un nouvel argument ? :grin: 
```suggestion
        update_fields=None,
        **kwargs,
```